### PR TITLE
New tutorial component

### LINKED ISSUE #3345: New tutorial component
**Is your feature request related to a problem? Please describe.**

Develop the new tutorial component based on Charlie's design:

image

### PR DESCRIPTION
**Pull request by mariopino**
Closes #3345 

**Description:**

Develop the new tutorial component based on Charlie's design:

image

Actual state:

image

NOTE: To see the tutorial go to **Portfolio Page.**

Thank you! 🚀

---

For contributor:

- [x] Added changes entries. Run `yarn changelog` for a guided process.
- [x] Reviewed `Files changed` in the github PR explorer
- [x] Attach screenshots of the UI components on the PR description (if applicable)
- [x] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/common/TmModalTutorial.vue [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<template>
+  <div
+    v-if="show"
+    v-focus-last
+    class="tm-modal-tutorial"
```
Comment (jbibla) [inline src/components/common/TmModalTutorial.vue]: no `tm` required. this is from Tendermint days. 😄

Comment (mariopino) [reply src/components/common/TmModalTutorial.vue]: All anachronic references removed

**Comment by faboweb** [comment]
@mariopino this looks dope and was done in record time! congrats!
Maybe hide this "demo" behind the experimental flag (store.session.experimentalMode)

**Comment by mariopino** [comment]
> @mariopino this looks dope and was done in record time! congrats!
> Maybe hide this "demo" behind the experimental flag (store.session.experimentalMode)

Yep, we can do that!

**Comment by mariopino** [comment]
This is done, additionally, there is a couple of things pending in this component (maybe for another issue/pr):

- Handle modal overlapping (only one modal at the same time in UI)
- Handle responsive/mobile behaviour

And more ideas (certainly for another pr/s :-) ):

- Get tutorials from the backend
- Store followed tutorials in localStorage to don't show again to user (showing a new unreaded one should be more interesting)

**Pull request merged**